e2e-tests/oauth-flow/rfc021/node-A/oauth2.js

```diff
@@ -0,0 +1,22 @@
+function introspectAccessToken(r) {
+    // strip the first 8 chars
+    var token = "token=" + r.headersIn['Authorization'].substring(7);
```

Comment:
does this always have the right encoding?

Reply:
we use 128bit b64 encoded strings, so for us: yes.

Reply:
I raw thinking of standard/raw/URL base64 encoding, but if it works...